### DISCUSSION TITLE
Template with examples.

### DISCUSSION THREAD

**Discussion by rinka88** [templaza/astroid-framework#542] [Ideas]
The new widgets are very cool and are being improved and added to with each update. Template two is also excellent, but there are very few examples there. I think it would be better to create a ready-made quickstart template with ready-made examples.

**Comment by ghost** [comment]
Hello, there is a very nice quickstart template from Template Two here:
https://github.com/templaza/astroid-framework/releases/tag/v3.0.9

Regards Olli

**Comment by rinka88** [comment]
I know about the “Two” template and where to download it, I installed it, but it doesn’t have all the Astroid features. I would like a template that shows all the features of the builder with the framework

**Comment by ghost** [comment]
Sorry, I just wanted to help.

**Comment by Chacapamac** [comment]
Yes, I’m working on it
